apps/passport-client/components/screens/FrogScreens/FrogHomeScreen.tsx

```diff
@@ -11,60 +19,31 @@ import { useSyncE2EEStorage } from "../../../src/useSyncE2EEStorage";
 import { MaybeModal } from "../../modals/Modal";
 import { AppContainer } from "../../shared/AppContainer";
 import { AppHeader } from "../../shared/AppHeader";
-import { PCDCard } from "../../shared/PCDCard";
 import { SyncingPCDs } from "../../shared/SyncingPCDs";
+import { ActionButton, Button } from "./Button";
+import { DexTab } from "./DEXTab";
+import { GetFrogTab } from "./GetFrogTab";
+import { ScoreTab } from "./ScoreTab";
 
 /** A placeholder screen for FrogCrypto.
  *
  * We might want to consider slotting this into the existing HomeScreen to better integrate with PCD explorer.
  */
 export function FrogHomeScreen() {
   useSyncE2EEStorage();
-  const dispatch = useDispatch();
   const syncSettled = useIsSyncSettled();
-  const { value: subs } = useSubscriptions();
-  const frogSub = useMemo(
-    () => subs.getActiveSubscriptions().find((sub) => sub.feed.name === "Bog"),
-    [subs]
-  );
+  const { subs, frogs, refetch } = useFrogSubscriptions();
+  const [tab, setTab] = useState<"get" | "score" | "dex">("get");
+
   const pcds = usePCDCollection();
   // NB: we cannot use useMemo because pcds are mutate in-place
-  const frogPCDs = pcds.getAllPCDsInFolder("FrogCrypto");
+  const frogPCDs = pcds
+    .getAllPCDsInFolder("FrogCrypto")
+    .filter(
+      (pcd): pcd is EdDSAFrogPCD => pcd.type === EdDSAFrogPCDPackage.name
```

Comment:
the function argument you're passing into this `filter` function should probably be declared in the frog pcd package in some sort of utils file, so that it can be used here.